src/apps/AppCenter/AppCenter.js

```diff
@@ -105,36 +105,53 @@ class AppCenter extends React.Component {
     const repos = this.getRepos()
     const currentRepo = openedRepoName && this.getRepoFromName(openedRepoName)
 
+    const navigationItems = [
+      'App Center',
+      ...(currentRepo ? [currentRepo.name] : []),
+    ]
+
     return (
       <React.Fragment>
         <AppView
+          style={{ height: '100%' }}
```

Comment:
I think we could set the [height prop](https://github.com/aragon/aragon-ui/blob/master/src/components/AragonApp/AppView.js#L18) instead of the style when we have to do this to `AppView`